src/testHelpers.tsx

```diff
@@ -17,3 +18,32 @@ export function renderWrapped(element: ReactElement): RenderResult {
     rerender: (newElement: ReactElement): void => renderResult.rerender(wrap(newElement)),
   };
 }
+
+export function aTestType(): TestType {
+  return {
+    id: 'some-test-type-id',
+    name: 'PCR test',
+    resultsSchema: {
```

Comment:
@StevePole Can you confirm this is the correct schema?

Reply:
Schema looks fine.  The `type: boolean` isn't strictly required, as the sub schema inside the oneOf dictates the type.  I'm 99% sure it's valid JSON schema, but could possibly confuse other JSON schema libraries.